usr/plugins/devel/caddy/src/opnsense/service/templates/Pischem/Caddy/Caddyfile

```diff
@@ -209,7 +212,15 @@
 # Reverse Proxy Domain: "{{ reverse['@uuid'] }}"
 {{ reverse.FromDomain|default("") }}{% if reverse.FromPort %}:{{ reverse.FromPort }}{% endif %} {
     {% if reverse.AccessLog|default("0") == "1" %}
+    {% if generalSettings.LogAccessPlain == "0" %}
```

Comment:
For error prevention it would be good to add it like this:

{% if generalSettings.LogAccessPlain|default("0")  == "0" %}

When there is no value, the default value will be used automatically. That prevents template reload errors if there is no value present (for any reason).